### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by OliverWieland** [jbox-web/redmine_git_hosting#403]
I tried to install redmine_git_hosting following this docuemntation:
http://jbox-web.github.io/redmine_git_hosting/howtos/install/#step-by-step-installation

I have checked twice that the checked out versions of redmine_bootrrap_kit and redmine_git_hosting are correct (0.2.2 and 1.0.2).

Regardless I get the following error message:
```
redmine@tbsw-bugtracker:~/redmine$ bundle install --without development test
Warning: this Gemfile contains multiple primary sources. Using `source` more than once without a block is a security risk, and may result in installing unexpected gems. To resolve this warning, use a block to indicate which gems should come from the secondary source. To upgrade this warning to an error, run `bundle config disable_multisource true`.
You cannot specify the same gem twice with different version requirements.
You specified: redcarpet (~> 2.3.0) and redcarpet (~> 3.1.2)
```

**Comment by OliverWieland** [comment]
Sorry for double post, I found the closed issue #393 right now.
But there is no solution for me either

**Comment by n-rodriguez** [comment]
> But there is no solution for me either

Depending on your Redmine version (2.x/3.x) you’ll have to comment/uncomment the right lines in the plugin’s ```Gemfile```.
